packages/react-next/src/components/SpinButton/SpinButton.base.tsx

```diff
@@ -0,0 +1,444 @@
+import * as React from 'react';
+import { IconButton } from '../../compat/Button';
+import { Label } from '../../Label';
+import { Icon } from '../../Icon';
+import {
+  getId,
+  KeyCodes,
+  calculatePrecision,
+  classNamesFunction,
+  precisionRound,
+  mergeAriaAttributeValues,
+  getNativeProps,
+  divProperties,
+} from '../../Utilities';
+import { getArrowButtonStyles } from './SpinButton.styles';
+import { ISpinButtonProps, ISpinButtonStyleProps, ISpinButtonStyles, KeyboardSpinDirection } from './SpinButton.types';
+import { Position } from 'office-ui-fabric-react/lib/utilities/positioning';
+import { KeytipData } from '../../KeytipData';
+import { useBoolean, useSetTimeout, useControllableValue } from '@uifabric/react-hooks';
+
+export interface ISpinButtonState {
```

Comment:
remove,
```suggestion
interface ISpinButtonVariables {
```

Reply:
1. "State" tends to refer to component state. but these in fact are not component state, but just local variables